api/src/organizational-entities/domain/errors.js

```diff
@@ -24,6 +24,14 @@ class AlreadyExistingOrganizationFeatureError extends DomainError {
   }
 }
 
+class DataProtectionOfficerEmailInvalid extends DomainError {
+  constructor({ code = 'DATA_PROTECTION_EMAIL_INVALID', message = 'Data protection email invalid', meta } = {}) {
```

Comment:
```suggestion
class DataProtectionOfficerEmailInvalid extends DomainError {
  constructor({ code = 'DATA_PROTECTION_OFFICER_EMAIL_INVALID', message = 'Data protection officer email invalid', meta } = {}) {
```

Mais on pourrait aussi plutôt mettre la forme suivante car dans le contexte métier du RGPD, le terme _DPO_ est plus utilisé que le terme  _DATA_PROTECTION_OFFICER_ (comme _URL_, cf. les standards de l'équipe), d'ailleurs c'est le terme _DPO_ qui est utilisé dans le format du fichier d'import :

```suggestion
class DpoEmailInvalid extends DomainError {
  constructor({ code = 'DPO_EMAIL_INVALID', message = 'DPO email invalid', meta } = {}) {
```